win/rl/winrl.cc

```diff
@@ -353,6 +360,50 @@ NetHackRL::fill_obs(nle_obs *obs)
             std::memset(obs->message, 0, NLE_MESSAGE_SIZE);
         }
     }
+
+    if (obs->menu_strs) {
+        bool found_menu = false;
+        for (int i = 0; i < windows_.size(); ++i) {
+            if (i != WIN_INVEN && windows_[i].get()->type == NHW_MENU) {
```

Comment:
I'll have to very carefully check the logic here. I'm especially surprised there's no `break` statement here?

Will get back to this.

Reply:
Yes - I'm not sure what to do if there are two windows - so I didn't put a break while testing to see if I could trigger it. If we go with a solution like this we should have a break.